indico/modules/events/papers/templates/paper.html

```diff
@@ -19,18 +21,27 @@
 {% block content %}
     {{ render_paper_info(paper) }}
 
-    {% call(block, revision) render_timeline(paper, comment_form, review_form) %}
+    {% call(block, revision) render_timeline(paper, comment_form, review_form, revision_form) %}
         {% if block == 'multiple-review-group-text' %}
             {{ render_multiple_review_group_text(paper) }}
         {% endif %}
         {% if block == 'revision-info' %}
             {{ render_submission_info(revision) }}
         {% endif %}
+        {% if block == 'revision-judgment' %}
+            {{ render_paper_judgment(revision) }}
+        {% endif %}
+        {% if block == 'submit-revision' %}
```

Comment:
But `block` cannot be `'revision-judgment'` and `'submit-revision'` at the same time...